### PR TITLE
737-i18n-to-all-views

### PR DESCRIPTION
**Pull request by wandergithub**
views->root->index

# 🔗 Issue

https://github.com/rubyforgood/pet-rescue/issues/737
# ✍️ Description

Implements I18n in views:
- views->root->index

# 📷 Screenshots/Demos

<details>
  <summary>views->root->index</summary>

  image

</details>

<details>
  <summary>views->layouts->shared:(all navbars  and footers on folder)</summary>

  image

</details>

### CODE REVIEW

File: app/views/root/index.html.erb [hunk 1]
```diff
@@ -3,35 +3,35 @@
...
-          <p class="lead mb-4">A free application for pet foster and adoption organizations who want to save time and manage their operations in one place.</p>
+          <h1 class="display-2 fw-bold mb-3"><%= t('.title') + " " %><u class="text-warning"><span class="text-primary"><%= t('.title_highlight') %></span></u></h1>
+          <p class="lead mb-4"><%= t('.description') %></p>
           <ul class="list-unstyled mb-5">
             <li class="mb-2">
               <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="var(--geeks-success)" class="bi bi-check-circle-fill" viewBox="0 0 16 16">
                 <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"></path>
               </svg>
-              <span class="ms-2">Easy to use</span>
+              <span class="ms-2"><%= t('.prod_cuality_1') %></span>
```
Comment (kasugaijin) [inline app/views/root/index.html.erb:13]: what does this mean?

File: config/locales/en.yml [hunk 2]
```diff
@@ -75,6 +75,31 @@ en:
       status: Status
       withdraw_submission: Withdraw Application
       confirm_withdraw: Withdraw your application
+  root:
+    index:
+      title: "Connecting hearts and homes"
+      title_highlight: "worldwide"
+      description: "A free application for pet foster and adoption organizations who want to save time and manage their operations in one place."
+      prod_cuality_1: "Easy to use"
```
Comment (kasugaijin) [inline config/locales/en.yml:83]: are these abbreviations? I am not familiar.

Comment (wandergithub) [reply config/locales/en.yml]: bit of Spanglish... my bad. Just added a better name for this.

**Review by kasugaijin — APPROVED** [review]
Thank you @wandergithub

**Pull request merged**